crawl-ref/source/dgn-overview.cc

```diff
@@ -25,11 +26,27 @@
 #include "religion.h"
 #include "scroller.h"
 #include "stairs.h"
+#include "store.h" //for level_id()
 #include "stringutil.h"
 #include "terrain.h"
 #include "travel.h"
 #include "unicode.h"
 
+enum AnnotationPossibilities
```

Comment:
this is very cosmetic but I would not choose this enum name, as it's not very descriptive, and we also don't tend to use camel case for enums. What about `nonbranch_annotations`?

Reply:
or something like `annotation_menu_commands`